src/styles/bloks/day.scss

```diff
@@ -0,0 +1,10 @@
+.calendar__day {
+  @include square($icon-size);
+  transition: 1s;
+
+  &:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+  }
+}
```

Comment:
you don't need a separate file for elements, only for blocks. it should be in the same file as its block `calendar`